homeassistant/components/systemmonitor/const.py

```diff
@@ -0,0 +1,8 @@
+"""Constants for systemmonitor."""
+from homeassistant.const import Platform
+
+CONF_ARG = "arg"
+CONF_INDEX = "index"
+DOMAIN = "systemmonitor"
+
+PLATFORMS = [Platform.SENSOR]
```

Comment:
Please move constants that are only used in a single module to that module. It makes it easier to read the module.